src/config.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+import os.path
+import re
+import secrets
+import string
+
+import jsonschema
+from src.ssm import get_parameter
+
+LOGGER = logging.getLogger("root")
+
+
+def schema_dir():
+    return os.path.dirname(os.path.dirname(os.path.abspath(__file__))) + "/schema/"
+
+
+def is_invalid(data):
+    if data is None:
+        return True
+
+    schema = load_config(schema_dir() + "config.json")
+    try:
+        jsonschema.validate(data, schema)
+    except jsonschema.exceptions.ValidationError as err:
+        return err
+    return False
+
+
+def load_config(filename):
+    if not is_config_exists(filename):
+        LOGGER.error("Provided configuration file is not available")
+        return None
+
+    with open(filename) as file:
+        data = json.load(file)
+
+    return data
+
+
+def is_config_exists(filename):
+    return os.path.isfile(filename)
+
+
+def is_sharing_enabled(data):
+    if data.get("Share") is None:
+        return False
+    return data["Share"]
+
+
+def _fetch_from_tfstate(key, tf_outputs):
+    if tf_outputs.get(key) is None:
```

Comment:
it'd make sense for me to call `tf_outputs.get(key)` and capture the value for return when it's not `None` instead of calling `tf_outputs[key]` later